docs/restore-individual-tables.md

```diff
@@ -1,76 +1,105 @@
+
+```{.bash data-prompt="mysql>"}
+mysql -u <username> -p -e "CREATE DATABASE create_database_test;"
+mysql> CREATE TABLE export_test (
 a int(11) DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 ```
 
+## Save the schema
+
+Before importing any table, you must save the schema definition for all tables you want to restore. Think of the schema as a blueprint for your table. XtraBackup only backs up data files, not these blueprints. Without the original schema, MySQL won't be able to use the restored .ibd files.
+
+When you back up your database, you may not know which tables to restore later. This operation prepares you for any future restoration needs. Also, table structures might change between the time when you create the backup and when you use mysqldump. Saving the most current structure helps ensure everything works during restoration.
+
+Review the script in [Dump schema for all tables](dump-schema.md) for an easier way to save schemas. You'll likely need to adjust this script to fit your setup.
+
+By saving schemas ahead of time, you ensure that all the table blueprints are ready when you start restoring from backups, making data recovery go more smoothly.
+
 ## Export the table
 
-To generate an `.ibd` file in the target directory, create the table using the `innodb_file_per_table` mode:
+First, create the table using the `innodb_file_per_table` setting. This setting tells MySQL to create a separate `.ibd` file for each table in your source directory. Run this command to find the created file in your system:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-During the `--prepare` step, add the [--export] option to the
-command. For example:
+Next, when you run the `--prepare` phase, add the [--export] option to your command. This option tells XtraBackup to create any extra files needed for moving the table:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-When restoring an encrypted InnoDB tablespace table, add the keyring file:
+If you're working with encrypted tables (ones that use transparent data encryption), you'll need extra security settings. You must include the keyring file that has the encryption keys:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/tmp/table \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
 ```
 
-The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the `.cfp` file, which contains the transfer key and an encrypted tablespace key.
-
-The files are located in the target directory:
+After these commands finish successfully, you'll need these files for the table import.
 
 ```text
 /data/backups/mysql/test/export_test.ibd
 /data/backups/mysql/test/export_test.cfg
+/data/backups/mysql/test/export_test.cfp
```

Comment:
since we are using t1, lets use t1 everywhere in this doc.

schema name/dir -> test
table ->t1

Reply:
or option 2: change the newl lines to export_test. trying to be consistent here. see down